Formula/ucg.rb

```diff
@@ -0,0 +1,33 @@
+class Ucg < Formula
+  desc "grep-like tool for searching large bodies of source code"
+  homepage "https://github.com/gvansickle/ucg"
+  url "https://github.com/gvansickle/ucg/releases/download/0.3.1/universalcodegrep-0.3.1.tar.gz"
+  sha256 "62f9ef88ea5c0777696c4322bed1fb9a3fb62eb85bd053af50f75d42ec259086"
+  head "https://github.com/gvansickle/ucg.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "automake" => :build
+  depends_on "argp-standalone" => :build
+  depends_on "pcre2"
+
+  # superenv's '-march' flag conflicts with build script
+  env :std
```

Comment:
Hi @equal-l2 ,

The 0.3.2 release is out in tarball form here: https://github.com/gvansickle/ucg/releases/tag/0.3.2
This release should not require the `env :std` to build under Homebrew.

Let me know how that tarball works; `travis-ci` is showing no problems on any of its OS X configurations, so I'm pretty confident this will go smoothly.

Thanks again @equal-l2 and everyone else,
GRVS